tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -66,15 +65,82 @@ applicable for the resource type.
 To view the list of your running resources:
 
 1. Select the Component from the Catalog index page.
-1. Select the Workloads tab.
+2. Select the Runtime Resources tab.
 
-    ![Workload index table](./images/workload-visibility-workloads.png)
+![Runtime resources index table collapsed](images/runtime-resources-index.png)
```

Comment:
might be helpful to show both screens: The catalog page highlighting one of the components in the catalog and another showing the runtime resources page with the runtime resources tab highlighted